duet/explores/mobile_ios_country.explore.lkml

```diff
@@ -0,0 +1,85 @@
+include: "../views/*.view.lkml"
+
+explore: mobile_ios_country {
+  label: "Mobile iOS Funnel"
```

Comment:
```suggestion
  label: "Mobile iOS Funnel"
  view_label: "iOS Funnel Steps"
```

that would apply to this name here:
<img width="314" alt="Screen Shot 2021-05-24 at 10 09 43 AM" src="https://user-images.githubusercontent.com/20819040/119360008-2bb03e00-bc78-11eb-982f-a8b1fd828fb5.png">